src/disco/containers/DiscoPane.js

```diff
@@ -172,7 +173,15 @@ export class DiscoPaneBase extends React.Component {
 
 export function loadedAddons(state) {
   return state.discoResults.map(
-    (result) => ({ ...result, ...state.addons[result.addon] })
+    (result) => {
+      return {
+        ...result,
+        // `result` comes from the API call in `src/disco/api.js` and
+        // normalizer makes everything complicated...
+        // `addon` is actually the add-on's GUID.
```

Comment:
I'd clarify with `result.addon` instead of `addon`, just a nitpick though.

Reply:
ah right, thanks